tests/models/edx/converters/xapi/test_open_response_assessment.py

```diff
@@ -0,0 +1,77 @@
+"""Tests for the open response assessment xAPI converters"""
+
+import json
+from uuid import UUID, uuid5
+
+import pytest
+from hypothesis import given, provisional, settings
+from hypothesis import strategies as st
+
+from ralph.models.converter import convert_dict_event
+from ralph.models.edx.converters.xapi.open_response_assessment import (
+    ORASaveSubmissionToQuestionSaved,
+)
+from ralph.models.edx.open_response_assessment import (
+    ORASaveSubmission,
+    ORASaveSubmissionEventField,
+    ORASaveSubmissionEventSavedResponseField,
+)
+from ralph.models.edx.x_block import ContextField, ContextModuleField
+
+
+@settings(max_examples=1)
+@given(
+    st.builds(
+        ORASaveSubmission,
+        referer=provisional.urls(),
+        context=st.builds(
+            ContextField,
+            user_id=st.just("1"),
+            path=st.just("https://fun-mooc.fr/"),
+            module=st.builds(ContextModuleField),
+        ),
+        event=st.builds(
+            ORASaveSubmissionEventField,
+            saved_response=st.builds(
+                ORASaveSubmissionEventSavedResponseField,
+                parts=st.just([{"text": "first"}, {"text": "second"}]),
+            ),
+        ),
+    ),
+    provisional.urls(),
+)
+@pytest.mark.parametrize("uuid_namespace", ["ee241f8b-174f-5bdb-bae9-c09de5fe017f"])
+def test_open_response_assessment_ora_save_submission_to_quesiont_saved_conversion(
+    uuid_namespace, event, home_page
+):
+    """Tests that converting with `ORASaveSubmissionToQuestionSaved` returns the expected value."""
+
+    event_str = event.json()
+    event = json.loads(event_str)
+    xapi_event = convert_dict_event(
+        event, event_str, ORASaveSubmissionToQuestionSaved(uuid_namespace, home_page)
+    )
+    xapi_event_dict = json.loads(xapi_event.json(exclude_none=True, by_alias=True))
+    assert xapi_event_dict == {
+        "id": str(uuid5(UUID(uuid_namespace), event_str)),
+        "actor": {
+            "account": {"homePage": home_page, "name": "1"},
+        },
+        "object": {
+            "definition": {
+                "name": {"en-US": "question"},
```

Comment:
Question regarding the `object.definition.name` field:
We could pass here the edx `context.module.display_name` field which holds the title of the ORA question but this title is not always in English and I don't see an easy solution to guess the used language. Do you prefer the static `question` value or the latter?

Reply:
It would be nice to have the original question! Should we assign a default language that could be overridden via a Ralph setting or a CLI option?

Reply:
Good Idea) If we could filter out events that are related to courses that are in a specific language prior the convert command and then pass the right language option - this could work.
We could automatize this by querying the mongodb `modulestore.structures` and `modulestore.active_versions` collections to get a dictionary with all course_keys (org+course_id+session_id) associated with their language.